### DISCUSSION TITLE
Start on logon

### DISCUSSION THREAD

**Discussion by MichalVildomec** [jonaskohl/CapsLockIndicator#163] [General]
Hello, 
Im trying to do an deployment of this app to some users, i made it that it will copy the indicator to the desktop and the appdata folder to app data, but now i need it that the startonlogon will move too,but it doesnt, on another pc the startonlogon is not checked and in the advanced settings there isnt event startonlogon that i can mark True, can u help me please? 

I tried to put it there manually but it doesnt work. 

So how do i achieve that even when i am moving the settings to another user pc the settings will stay? all of the other settings stay, the only one that doesnt is the startonlogon.

**Comment by jonaskohl** [comment]
CapsLock Indicator uses the Windows Registry to manage its autorun function. It creates a `REG_SZ` value named `CapsLock Indicator` at `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Run`. See [`MainForm.cs:135`](https://github.com/jonaskohl/CapsLockIndicator/blob/master/CapsLockIndicatorV3/MainForm.cs#L135) and [`MainForm.cs:1206`](https://github.com/jonaskohl/CapsLockIndicator/blob/master/CapsLockIndicatorV3/MainForm.cs#L1206) for reference.

**Comment by MichalVildomec** [comment]
Thanks so much, will try

From: Jonas Kohl ***@***.***>
Sent: Monday, June 26, 2023 8:17 PM
To: jonaskohl/CapsLockIndicator ***@***.***>
Cc: VILDOMEC Michal ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [jonaskohl/CapsLockIndicator] Start on logon (Discussion #163)

CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe!



CapsLock Indicator uses the Windows Registry to manage its autorun function. It creates a REG_SZ named CapsLock Indicator value at HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Run. See MainForm.cs:135<https://github.com/jonaskohl/CapsLockIndicator/blob/master/CapsLockIndicatorV3/MainForm.cs#L135> and MainForm.cs:1206<https://github.com/jonaskohl/CapsLockIndicator/blob/master/CapsLockIndicatorV3/MainForm.cs#L1206> for reference.
